### ISSUE TITLE
Sidebar not focusable with keyboard

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1592]
The sidebar (Explore, Featured, Productivity…) is not focusable with the keyboard. This means it is not usable to anyone not using a mouse, and is inaccessible to Ubuntu's built-in screen reader.
